sparse/sparse_array.py

```diff
@@ -212,3 +212,19 @@ def __array__(self, **kwargs):
                                'To manually densify, use the todense method.')
 
         return np.asarray(self.todense(), **kwargs)
+
+    def __array_function__(self, func, types, args, kwargs):
```

Comment:
You'll want to move the changes over here... My bad for suggesting them in the other place.

Reply:
Yup, already committed that locally. No worries, I had too many implementations, so that was my fault. :)

Reply:
Just working on some tests now, then I'll push everything at once.